ratelimits/limiter_test.go

```diff
@@ -38,6 +40,12 @@ func newTestTransactionBuilder(t *testing.T) *TransactionBuilder {
 }
 
 func setup(t *testing.T) (context.Context, map[string]*Limiter, *TransactionBuilder, clock.FakeClock, string) {
+	// Because the config dir will not change during this process' life, we
+	// don't need to call features.Reset().
```

Comment:
I quibble with this statement. Here's what's going on here, which I think would make a better comment:

Because all test cases in this file are affected by this feature flag, we want to run them all both with and without the feature flag. One handy way to do this is to set the feature flag in `setup`, and set it or not set it based on `BOULDER_CONFIG_DIR`, so we get one set of runs with and without. It's difficult to `defer features.Reset()` from the setup func, and so long as this code doesn't test any other features we don't need to. But this is fragile: if any of the test cases _does_ call `features.Reset()`, it will not be testing the intended code path. But we expect to clean this up quickly so it's alright.